decidim-admin/app/views/decidim/admin/organization_appearance/form/_colors.html.erb

```diff
@@ -1,6 +1,16 @@
 <div class="card">
   <div class="card-divider">
-    <h2 class="card-title"><%= t ".colors_title" %></h2>
+    <h2 class="card-title"><%= t ".colors_title" %> </h2>
+  </div>
+  <div class="callout warning" data-closable>
+    <svg role="img" class="icon--warning icon">
+      <title></title>
+    </svg>
+    <p><%= t ("from.colors.warning_colors") %> </p>
+    <a href="https://webaim.org/resources/contrastchecker">WebAIM Contrast Checker</a>
+    <button class="close-button" aria-label="Dismiss alert" type="button" data-close>
```

Comment:
In this particular case, I think it'd be better that this notice hasn't the close capability.